src/cart/crt_init.c

```diff
@@ -13,48 +13,11 @@
 #include "crt_internal.h"
 
 struct crt_gdata	crt_gdata;
+struct crt_envs_t       g_envs;
```

Comment:
we should be consistent and prefix global variables with crt_, even more if it's not static.

Reply:
ok